setup.py

```diff
@@ -112,6 +112,9 @@ def run(self, *arg, **kw):
 extrasReqs['sftp'] = [
     'paramiko',
 ]
+extrasReqs['mount'] = [
```

Comment:
Does it make more sense to name this extra "fuse" instead of "mount"?

Reply:
My thought was the dependency is for the mount command.  Just like we name the paramiko dependency sftp, since that it what it is used with.

Reply:
Well, SFTP is the capability, paramiko is an implementation detail. I don't understand enough about FUSE to say whether this is appropriate -- could there ever be a "mount" context that doesn't involve "fuse"?

Reply:
I was trying to match conceptually.  There are different python fuse libraries.  And, I suppose, one could write a kernel driver rather than use fuse at all.

Reply:
Ok, fair enough. I'm fine with `mount`.